### PR TITLE
add server capabilities to owncloudcmd

### PR DESCRIPTION
**Pull request by mrow4a**
@owncloud/desktop-developers

### CODE REVIEW

**Comment by dragotin** [comment]
please at least ifdef the C++ 11 stuff to keep it compileable with older systems and/or Qt4

File: src/cmd/cmd.cpp [hunk 1]
```diff
@@ -68,6 +69,10 @@ struct CmdOptions {
 // So we have to use a global variable
 CmdOptions *opts = 0;
 
+// Make sure the timeout for this job is less than how often we get called
+// This makes sure we get tried often enough without "ConnectionValidator already running"
+static qint64 timeoutToUseMsec = qMax(1000, ConnectionValidator::DefaultCallingIntervalMsec - 5*1000);
```
Comment (ogoffart) [inline src/cmd/cmd.cpp]: should be const

(to avoid runtime initialisation)

File: src/cmd/cmd.cpp [hunk 2]
```diff
@@ -68,6 +69,10 @@ struct CmdOptions {
 // So we have to use a global variable
 CmdOptions *opts = 0;
 
+// Make sure the timeout for this job is less than how often we get called
```
Comment (ogoffart) [inline src/cmd/cmd.cpp]: this comment does not make sens here.

**Comment by ogoffart** [comment]
Looks good, thanks   :+1: :

**Comment by guruz** [comment]
👍 Please merge

**Comment by guruz** [comment]
Next time format the message like:
```owncloudcmd: Add server capability check```

**Comment by rullzer** [comment]
Ah that would be usefull when debugging indeed. :+1:

**Pull request merged**